dns/dyndns/src/etc/inc/plugins.inc.d/dyndns/phpDynDNS.inc

```diff
@@ -383,6 +390,7 @@ class updatedns
      */
     function _update()
     {
+        $status_intro = 'Dynamic DNS';
```

Comment:
I cleaned up use of this missing var, although it's duplication it doesn't hurt to embed this into the target strings.

Reply:
Yes, I cleaned it up then.
It's directly in the strings.